infrastructure/lambda/template.yaml

```diff
@@ -253,6 +253,31 @@ Mappings:
 
 
 Resources:
+  AddressTableKey:
+    Type: AWS::KMS::Key
+    Properties:
+      Description: !Sub "${AWS::StackName} address table key"
+      Enabled: true
+      KeySpec: SYMMETRIC_DEFAULT
+      KeyUsage: ENCRYPT_DECRYPT
+      MultiRegion: false
+      PendingWindowInDays: 30
```

Comment:
I chose to omit these in Check as they are the default values (based on the [docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-resource-kms-key.html)) but that's not a strong opinion